doc/src/main/asciidoc/inc/_enricher.adoc

```diff
@@ -92,6 +92,9 @@ fabric8-maven-plugin comes with a set of enrichers which are enabled by default.
 
 | <<fmp-maven-issue-mgmt-enricher>>
 | Add Maven Issue Management information as annotations to the kubernetes/openshift resources
+
+| <<f8-revision-history-enricher>>
```

Comment:
actually we used 'fmp' as prefix for standard enrichers and 'f8' for fabric8 specific enrichers. Although as I just see this convention is not really followed reigorously (also I found some generic enricher in f8, too), I'd like to stick to this convention for new enrichers.

For v4 we should go over all enrichers and check how we can categorize them better.